Remotion/ObjectBinding/Web.ClientScript/Scripts/BocList.ts

```diff
@@ -104,43 +104,44 @@ class BocList
   //  hasClickSensitiveRows: true if the click event handler is bound to the data rows.
   //  onSelectionChangedHandler: A function to be invoked when the BocList's selection changes. 
   //                             First argument is the BocList, second argument is a flag indicating whether the callback is invoked during initialization.
-  public static InitializeList(bocList: HTMLElement, selectRowSelectorControlName: string, selectAllSelectorControlName: string, selection: number, hasClickSensitiveRows: boolean, onSelectionChangedHandler: BocList_SelectionChangedHandler): void
+  public static InitializeList(bocListOrSelector: CssSelectorOrElement<HTMLElement>, selectRowSelectorControlName: string, selectAllSelectorControlName: string, selection: number, hasClickSensitiveRows: boolean, onSelectionChangedHandler: BocList_SelectionChangedHandler): void
   {
+    ArgumentUtility.CheckNotNull("bocListOrSelector", bocListOrSelector);
+    const bocList = ElementResolverUtility.ResolveSingle(bocListOrSelector);
+
     if (BocList.HasDimensions (bocList))
     {
-      $(bocList).addClass('hasDimensions');
-      BocList.FixUpScrolling($(bocList));
+      bocList.classList.add('hasDimensions')
+      BocList.FixUpScrolling(bocList);
     }
 
     var selectedRows = new BocList_SelectedRows (selection);
     if (   selectedRows.Selection != BocList._rowSelectionUndefined
         && selectedRows.Selection != BocList._rowSelectionDisabled)
     {
-      selectedRows.SelectRowSelectorControls = $('input[name="' + selectRowSelectorControlName + '"]');
-      selectedRows.SelectAllSelectorControls = $('input[name="' + selectAllSelectorControlName + '"]');
+      selectedRows.SelectRowSelectorControls = document.querySelectorAll<HTMLInputElement>('input[name="' + selectRowSelectorControlName + '"]')!;
+      selectedRows.SelectAllSelectorControls = document.querySelectorAll<HTMLInputElement>('input[name="' + selectAllSelectorControlName + '"]')!;
       selectedRows.OnSelectionChanged = onSelectionChangedHandler;
 
-      selectedRows.SelectRowSelectorControls.each (function (this: JQuery)
+      selectedRows.SelectRowSelectorControls.forEach (checkBox =>
       {
         selectedRows.DataRowCount++;
 
-        var checkBox = this;
-
-        var tableCell = checkBox.parentNode;
+        var tableCell = checkBox.parentNode!;
         if (tableCell.nodeName !== 'TD')
           throw 'Unexpected element type: \'' + tableCell.nodeName + '\'';
 
-        var tableRow = tableCell.parentNode;
+        var tableRow = tableCell.parentNode as HTMLElement;
         if (tableRow.nodeName !== 'TR')
           throw 'Unexpected element type: \'' + tableRow.nodeName + '\'';
 
         if (hasClickSensitiveRows)
-          BocList.BindRowClickEventHandler (bocList, tableRow, tableCell, this);
+          BocList.BindRowClickEventHandler (bocList, tableRow as HTMLElement, tableCell as HTMLElement, checkBox);
```

Comment:
...and then we can remove the casts from here.

```suggestion
          BocList.BindRowClickEventHandler (bocList, tableRow, tableCell, checkBox);
```